cassandra_migrate/migrator.py

```diff
@@ -377,8 +391,15 @@ def _create_version(self, version, migration):
             version, migration))
 
         version_id = uuid.uuid4()
+
+        # Cassandra versions below 2.2 use protocol 3 and use `dateof` instead
+        # of `toTimestamp`
+        if self.cluster.protocol_version == 3:
```

Comment:
Should this be `<= 3`?

Reply:
Or could we just use `dateOf` directly? I'm not aware of the differences between that and `toTimestamp`, but the CQL docs don't mention any.

Reply:
`dateOf` is deprecated, so should only be used in old versions (where it wasn't deprecated). And you are right about `<= 3`, I don't know why I didn't do it.